src/primitives.ts

```diff
@@ -22,6 +22,10 @@ export class Address implements IAddress {
     bech32(): string {
         return this.value;
     }
+
+    toString() {
+        return this.value;
```

Comment:
identical with the `bech32()` function. maybe return `this.bech32()` for code readability?

Reply:
Fixed.